spec/integration/hanami/middleware/body_parser_spec.rb

```diff
@@ -6,8 +6,22 @@
 RSpec.describe Hanami::Middleware::BodyParser do
   let(:app) { ->(_env) { [200, {}, "app"] } }
 
-  context "unknown parser" do
-    it "raises error" do
+  describe ".new" do
+    it "accepts a parser identificator" do
+      body_parser = Hanami::Middleware::BodyParser.new(app, :json)
+
+      expect(body_parser.instance_variable_get("@parsers")["application/json"])
+        .to be_instance_of(Hanami::Middleware::BodyParser::JsonParser)
+    end
+
+    it "accepts a parser identificator with additional mime-type(s)" do
+      body_parser = Hanami::Middleware::BodyParser.new(app, [json: "application/json+scim"])
```

Comment:
Do we actually want two specs here?

1. Like this one, which shows the `json` parser being used with exactly 1 additional mime type (not wrapped in an array), and
2. Another one that shows the `json` parser being used with >1 mime type (which requires them to be wrapped in an array)

Otherwise we have nothing exercising the Array() wrapping code up in `#build_parsers`.

(I know this might sound picky, but I genuinely think it'd be helpful, since I was hoping the specs might show me how the `parser_specs` value was meant to work!)

Reply:
@timriley great catch, this was actually broken